tensorboard/webapp/metrics/views/main_view/common_selectors_test.ts

```diff
@@ -65,6 +79,114 @@ describe('common selectors', () => {
         metrics: null,
       },
     };
+
+    runTableItems = [
+      {
+        run: {
+          id: 'run1-id',
+          name: 'run1',
+          startTime: 0,
+          hparams: [
```

Comment:
I am not sure why RunTableItem has hparam array inside the Run and also has hparam Map at the next level up. We never use this value in any of these test do we? Can you just make this null to make that clear?

Reply:
I've set this to `null`